File: GestionAeropuerto.API/Models/Vuelo.cs
namespace GestionAeropuerto.API.Models
{
    public class Vuelo
    {
        public int VueloID { get; set; }
        public string NumeroVuelo { get; set; } = string.Empty;
        public int AerolineaID { get; set; }
        public int AeronaveID { get; set; }
        public string Origen { get; set; } = string.Empty;
        public string Destino { get; set; } = string.Empty;
        public DateTime FechaSalida { get; set; }
        public DateTime FechaLlegada { get; set; }
        public string Estado { get; set; } = "Programado";
        public int? PuertaID { get; set; }
        public int AsientosDisponibles { get; set; }
        public decimal PrecioBase   { get; set; }
        public DateTime FechaRegistro { get; set; }

        public string? NombreAerolinea { get; set; }
        public string? ModeloAeronave { get; set; } 
        public string? NumeroPuerta { get; set; }   

    }
}

File: GestionAeropuerto.VB/Vuelo.vb
Public Class Vuelo
    Public Property VueloID As Integer
    Public Property NumeroVuelo As String
    Public Property AerolineaID As Integer
    Public Property NombreAerolinea As String
    Public Property AeronaveID As Integer
    Public Property ModeloAeronave As String
    Public Property Origen As String
    Public Property Destino As String
    Public Property FechaSalida As DateTime
    Public Property FechaLlegada As DateTime
    Public Property Estado As String
    Public Property PuertaID As Integer?
    Public Property NumeroPuerta As String
    Public Property AsientosDisponibles As Integer
    Public Property PrecioBase As Decimal

    Public Sub New()
        NumeroVuelo = ""
        NombreAerolinea = ""
        ModeloAeronave = ""
        Origen = ""
        Destino = ""
        Estado = "Programado"
        NumeroPuerta = ""
    End Sub
End Class